html/admin/api/projects/getAuditLog.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * API
+ * \projects\getAuditLog.php
+ * get audit log data for given project
+ *
+ * Arguments:
+ *  - projects_id: a project
+ */
+
+require_once __DIR__ . '/../apiHeadSecure.php';
+
+if (!$AUTH->instancePermissionCheck(20) or !isset($_POST['projects_id'])) finish(false);
+
+$DBLIB->where("auditLog.auditLog_deleted", 0);
+$DBLIB->where("auditLog.projects_id", $_POST['projects_id']);
+$DBLIB->where("auditLog.auditLog_actionTable", "projects");
+$DBLIB->join("users", "auditLog.users_userid=users.users_userid", "LEFT");
+$DBLIB->orderBy("auditLog.auditLog_timestamp", "DESC");
+$DBLIB->orderBy("auditLog.auditLog_id", "DESC");
+$auditLog = $DBLIB->get("auditLog",null, ["auditLog.*", "users.users_name1", "users.users_name2", "users.users_email"]);
+
```

Comment:
This is a huge amount of data for some projects, because it contains every action for the project - if you're not displaying it all could we jsut filter by comments?
I'm concerned by the volume of data we're trying to download on a mobile connection

Reply:
I suppose this depends on whether this API is only targeting a mobile app, or is going to replace all backend PHP logic down the line.

Think we need to clarify this in the roadmap, but am happy to convert to a comments only endpoint as that's what I want with it.

Alternatively would another argument for filtering what content is sent make sense?